README.md

```diff
@@ -80,6 +80,9 @@ Install dependencies:
 npm install
 ```
 
+Setup Environment Variables:
+Copy and paste env.sample into your .env file
```

Comment:
How about using [a ".env" file in Docker Compose](https://docs.docker.com/compose/environment-variables/#the-env-file)?

Reply:
I honestly don't know much at all about Docker -- Do you mean that we should use environment variables in Docker Compose for default variables that can be overwritten by an .env file?